[2022-07-11 09:17:36,882] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAD.ingest scheduled__2022-04-05T06:00:00+00:00 [queued]>
[2022-07-11 09:17:36,913] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAD.ingest scheduled__2022-04-05T06:00:00+00:00 [queued]>
[2022-07-11 09:17:36,914] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-11 09:17:36,914] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-11 09:17:36,915] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-11 09:17:36,980] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): ingest> on 2022-04-05 06:00:00+00:00
[2022-07-11 09:17:36,991] {standard_task_runner.py:52} INFO - Started process 8222 to run task
[2022-07-11 09:17:36,997] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAD', 'ingest', 'scheduled__2022-04-05T06:00:00+00:00', '--job-id', '119', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_postgres.py', '--cfg-path', '/tmp/tmp4h8c9sm9', '--error-file', '/tmp/tmp_vvqqsgh']
[2022-07-11 09:17:36,999] {standard_task_runner.py:80} INFO - Job 119: Subtask ingest
[2022-07-11 09:17:37,186] {task_command.py:370} INFO - Running <TaskInstance: LocalIngestionDAD.ingest scheduled__2022-04-05T06:00:00+00:00 [running]> on host 3dadab756f88
[2022-07-11 09:17:37,614] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAD
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-04-05T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-05T06:00:00+00:00
[2022-07-11 09:17:37,619] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-11 09:17:37,622] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls']
[2022-07-11 09:17:37,661] {subprocess.py:85} INFO - Output:
[2022-07-11 09:17:37,669] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-11 09:17:37,756] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAD, task_id=ingest, execution_date=20220405T060000, start_date=20220711T091736, end_date=20220711T091737
[2022-07-11 09:17:37,854] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-11 09:17:37,952] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-07-12 08:42:46,352] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAD.ingest scheduled__2022-04-05T06:00:00+00:00 [queued]>
[2022-07-12 08:42:46,374] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAD.ingest scheduled__2022-04-05T06:00:00+00:00 [queued]>
[2022-07-12 08:42:46,376] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-12 08:42:46,377] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-12 08:42:46,377] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-12 08:42:46,407] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): ingest> on 2022-04-05 06:00:00+00:00
[2022-07-12 08:42:46,416] {standard_task_runner.py:52} INFO - Started process 1392 to run task
[2022-07-12 08:42:46,421] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAD', 'ingest', 'scheduled__2022-04-05T06:00:00+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_postgres.py', '--cfg-path', '/tmp/tmp88xd6q5l', '--error-file', '/tmp/tmpyxyco36q']
[2022-07-12 08:42:46,423] {standard_task_runner.py:80} INFO - Job 25: Subtask ingest
[2022-07-12 08:42:46,554] {task_command.py:370} INFO - Running <TaskInstance: LocalIngestionDAD.ingest scheduled__2022-04-05T06:00:00+00:00 [running]> on host ef58f039b0e2
[2022-07-12 08:42:46,682] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2022-07-12 08:42:46,800] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAD
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-04-05T06:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-04-05T06:00:00+00:00
[2022-07-12 08:42:46,815] {logging_mixin.py:115} INFO - <sqlalchemy.engine.base.Connection object at 0x7f88c9a109d0>
[2022-07-12 08:42:46,815] {python.py:173} INFO - Done. Returned value was: None
[2022-07-12 08:42:46,842] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAD, task_id=ingest, execution_date=20220405T060000, start_date=20220712T084246, end_date=20220712T084246
[2022-07-12 08:42:46,875] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-12 08:42:46,962] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
